examples/webhook.py

```diff
@@ -0,0 +1,13 @@
+import os
+from aiotg import Bot
+
+bot = Bot(api_token=os.environ["API_TOKEN"])
+bot.set_webhook(url='https://yourserver.com/webhook/fdhgufdgh98498gvneriug489')
+
+
+@bot.command(r"/echo (.+)")
+def echo(chat, match):
+    return chat.reply(match.group(1))
+
+
+bot.run()
```

Comment:
And this line will look like `bot.run(webhook='https://yourserver.com/webhook/fdhgufdgh98498gvneriug489')`